src/main/java/mate/academy/rickandmorty/controller/ApiController.java

```diff
@@ -0,0 +1,18 @@
+package mate.academy.rickandmorty.controller;
+
+import lombok.RequiredArgsConstructor;
+import mate.academy.rickandmorty.model.CharacterEntity;
+import mate.academy.rickandmorty.service.RickAndMortyClient;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequiredArgsConstructor
+public class ApiController {
+    private final RickAndMortyClient rickAndMortyClient;
```

Comment:
you need client only for fetching data from open api, in your case only for bootstraping data in DB on startup. use services for other purposes, in this case - work with your DB
```suggestion
    private final CharacterService characterService;
```